paddle/go/pserver/service.go

```diff
@@ -135,7 +138,10 @@ func (s *Service) SendGrads(grads []Gradient, dummy *int) error {
 	errCh := make(chan error, count)
 	for _, g := range grads {
 		go func(p Parameter, g Gradient) {
```

Comment:
Thanks for your kindly reminding!  I'm agreed. I thought that we don't need to protect against concurrent update.  
I remember there is some theory bound on that topic, 
such as Eric Xing's **latency bounded sgd**, some async sgd. published by google,  I'am not quite sure it has effect in the learning performance.(maybe you are more familiar with that than me).